spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/autoconfig/DefaultCloudSqlJdbcInfoProvider.java

```diff
@@ -36,27 +35,21 @@
 	public DefaultCloudSqlJdbcInfoProvider(GcpCloudSqlProperties properties) {
 		this.properties = properties;
 		Assert.hasText(this.properties.getDatabaseName(), "A database name must be provided.");
-		if (StringUtils.isEmpty(this.properties.getJdbcUrl())) {
-			Assert.hasText(this.properties.getInstanceConnectionName(),
-					"An instance connection name must be provided. Refer to "
-							+ GcpCloudSqlAutoConfiguration.INSTANCE_CONNECTION_NAME_HELP_URL
-							+ " for more information.");
-		}
+		Assert.hasText(this.properties.getInstanceConnectionName(),
+				"An instance connection name must be provided. Refer to "
+						+ GcpCloudSqlAutoConfiguration.INSTANCE_CONNECTION_NAME_HELP_URL
+						+ " for more information.");
 	}
 
 	@Override
 	public String getJdbcDriverClass() {
-		return StringUtils.isEmpty(this.properties.getJdbcDriver())
```

Comment:
Also, why removing this?
Users must be able to specify a custom JDBC driver through properties.